src/borkdude/deps.clj

```diff
@@ -588,9 +601,10 @@ For more info, see:
          (when (.exists tools-jar) (.getPath tools-jar))
          (binding [*out* *err*]
            (warn "Clojure tools not yet in expected location:" (str tools-jar))
-           (clojure-tools-jar-download libexec-dir (vec (concat clj-jvm-opts
-                                                                proxy-settings
-                                                                ["-Xms256m"])))
+           (let [java-clj-jvm-opts (when clj-jvm-opts (vec (concat clj-jvm-opts
+                                                                   proxy-settings
+                                                                   ["-Xms256m"])))]
```

Comment:
What is the -Xms256m argument for?

Reply:
I picked it up from the `clj-main-cmd` a little bit further down the code, I assumed it was for some unknown to me use, shall I remove it?